src/javascript/_common/utility.js

```diff
@@ -229,6 +233,7 @@ module.exports = {
     template,
     isEmptyObject,
     cloneObject,
+    unique,
```

Comment:
I think this should be on line 237, we seem to be sorting by function placement in this file